### PR TITLE
Replace Recipe Name to Collection ID.

### PR DESCRIPTION
**Pull request by junaruga**
I would like to send PR to your PR branch to replace Recipe Name to Collection ID.
The reason is here https://github.com/sclorg/rpm-list-builder/commit/4adce90 .

### CODE REVIEW

File: rpmlb/cli.py [hunk 1]
```diff
@@ -75,14 +75,14 @@
     'recipe_file',
     type=click.Path(exists=True, dir_okay=False, resolve_path=True),
 )
-@click.argument('recipe_name')
-def run(recipe_file, recipe_name, **option_dict):
-    """Download and build RPMs listed in RECIPE_FILE under RECIPE_NAME
-    (such as 'python33').
+@click.argument('collection_id')
+def run(recipe_file, collection_id, **option_dict):
+    """Download and build RPMs listed on COLLECTION_ID (such as 'python33')
```
Comment (khardix) [inline rpmlb/cli.py]: I believe "under" is the right word to use here (... listed *under* COLLECTION_ID ...).

**Review by khardix — CHANGES_REQUESTED** [review]
Please fix the wording in the help message. Other than that, no issues from me.

**Comment by khardix** [comment]
Sorry, I force-pushed the original commit… Please rebase on to of that, thanks.

File: rpmlb/cli.py (hunk 1 above)
Comment (junaruga) [reply rpmlb/cli.py]: OK I will change to your way.

**Comment by junaruga** [comment]
OK Rebased and Pushed.

**Review by khardix — APPROVED** [review]

**Pull request merged**